Gemfile

```diff
@@ -41,16 +41,16 @@ gem 'omniauth-cas'
 gem 'omniauth-saml'
 
 # Honeycomb
-gem 'sequel'
-gem 'honeycomb-beeline'
-gem 'rack-honeycomb', '~> 0.5.0'
+#gem 'sequel'
```

Comment:
If we're cutting gems, lets try not to just comment them out.
We discussed not wanting to see code behind comments as much as possible during standups, 8/5.

Reply:
:+1: